From: Iceyer <iceyers@gmail.com>
Date: Fri, 21 Jul 2017 14:43:24 +0800
Subject: Add ordered config
Forwarded: https://code.launchpad.net/~iceyers/gsettings-qt/fix-link-path

This patch would fix random FTBFS when doing parallel build after
debhelper v10 due to tests built before real libraries.

---
 gsettings-qt.pro | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/gsettings-qt.pro gsettings-qt.pro
index 15e3684..ee04310 100644
--- a/gsettings-qt.pro
+++ gsettings-qt.pro
@@ -1,2 +1,7 @@
 TEMPLATE = subdirs
-SUBDIRS += src/gsettings-qt.pro GSettings/gsettings-qt.pro tests/tests.pro tests/cpptest.pro
+CONFIG += ordered
+SUBDIRS += \
+    src/gsettings-qt.pro \
+    GSettings/gsettings-qt.pro \
+    tests/tests.pro \
+    tests/cpptest.pro
